### PR TITLE
Add tests for csv_info.go

### PR DESCRIPTION
**Pull request by sebrandon1**
Adding some more coverage for the autodiscover/csv_info functions.

### CODE REVIEW

File: pkg/config/autodiscover/csv_info_test.go [hunk 1]
```diff
@@ -57,3 +58,106 @@ func TestCSVGetAnnotationValue(t *testing.T) {
...
+		expectedString string
+	}{
+		{
+			expectedString: "error (this is an error) attempting to unmarshal value of annotation 'testKey' on CSV 'testnamespace/testname'",
+		},
+	}
+
+	for _, tc := range testCases {
+		csvr := CSVResource{
+			Metadata: struct {
```
Comment (greyerof) [inline pkg/config/autodiscover/csv_info_test.go:73]: This is not important, but I'm curious: is it necessary to re-declare the whole struct types for the initialization? I'd just do this:
```
csvr := CSVResource{}
csvr.Name = "testname"
csvr.Namespace = "testnamespace"
```

File: pkg/config/autodiscover/csv_info_test.go [hunk 2]
```diff
@@ -57,3 +58,106 @@ func TestCSVGetAnnotationValue(t *testing.T) {
...
+		origFunc := executeOcGetAllCommand
+		executeOcGetCommand = func(resourceType, labelQuery, namespace string) string {
+			output, err := os.ReadFile(path.Join(filePath, tc.filename))
+			assert.Nil(t, err)
+			return string(output)
+		}
+
+		outputList, err := GetCSVsByLabelByNamespace(tc.label, tc.value, "testnamespace")
+		assert.Nil(t, err)
+		if len(outputList.Items) > 0 {
```
Comment (greyerof) [inline pkg/config/autodiscover/csv_info_test.go:157]: I believe that the number of parsed Items should also be checked with something like:
`assert.Equal(t, len(outputList.Items), tc.ExpectedItemsCount)`

File: pkg/config/autodiscover/csv_info_test.go (hunk 1 above)
Comment (sebrandon1) [reply pkg/config/autodiscover/csv_info_test.go:73]: Yeah that was an interesting auto-fill by `gofmt`.  Not sure why it brought all of that in.  I can double check if removing it causes any issues.

File: pkg/config/autodiscover/csv_info_test.go (hunk 2 above)
Comment (sebrandon1) [reply pkg/config/autodiscover/csv_info_test.go:157]: Good idea, I'll add an expected items count.

File: pkg/config/autodiscover/csv_info_test.go (hunk 1 above)
Comment (sebrandon1) [reply pkg/config/autodiscover/csv_info_test.go:73]: Nope, it doesn't like it when I remove it.

File: pkg/config/autodiscover/csv_info_test.go (hunk 2 above)
Comment (sebrandon1) [reply pkg/config/autodiscover/csv_info_test.go:165]: Done.

**Review by jun-rh — APPROVED** [review]

**Review by greyerof — APPROVED** [review]

**Pull request merged**